mlir-sycl/doc/rebasing.md

```diff
@@ -0,0 +1,62 @@
+# Rebasing `sycl` Branch
+
+The SYCL-MLIR project is developed in the [`sycl-mlir` branch of the `intel/llvm`
+repository](https://github.com/intel/llvm/tree/sycl-mlir). As it is intended to be a SYCL front-end,
+it should be tracking the [`sycl` branch of that
+repository](https://github.com/intel/llvm/tree/sycl). Therefore, establishing a process to keep up
+with `sycl` progress in `sycl-mlir` is key to success.
+
+## Pulldown
```

Comment:
These instructions are based on the setup described [here](https://github.com/intel/llvm/blob/sycl-mlir/mlir-sycl/doc/developer/ContributeToSYCLMLIR.md), we should include a link to this document here.